### DISCUSSION TITLE
help im new to this

### DISCUSSION THREAD

**Discussion by zipporah2145** [vercel/community#5467] [Help]
### Summary

im not even sure how to make a web proxy that can acces astroid.gg as well as now. gg for roblox can someone walk me through the steps or just anything please man

### Steps to Reproduce

i just want my own proxy to play roblox on school computer help!

**Comment by Mtigee** [comment]
English pls

**Comment by Mtigee** [comment]
Simply download / copy miniProxy.php to your web server (it's okay to rename it) and access it directly. That's it! You'll be presented with further usage instructions.

miniProxy doesn't require any configuration out of the box, but configuration options are available; see the top of miniProxy.php for details.
